src/applications/mhv/landing-page/utilities/index.js

```diff
@@ -0,0 +1,16 @@
+function getDestinationDomain(href) {
+  const url = new URL(href, document.location.href);
+
+  const { hostname } = url;
+
+  if (hostname.includes('vagov')) {
```

Comment:
I'm still not clear on when the hostname will include `'vagov'` or `'mhvnp'`. Are those used in lower environments? I don't think I ever expect to see those strings in a hostname or URL..

Reply:
`if (hostname.includes('myheatlh.va.gov') return "mhvnp";`
`if (hostname.includes('www.va.gov') return "vagov';

Reply:
I fixed the destination domain function to be more flexible. It captures the hostname without regard for whether it is vagov or the other one